src/platform/i386/exception.c

```diff
@@ -5,6 +5,7 @@
 #include "string.h"
 #include "isr.h"
 #include "chal_cpu.h"
+#include "chal_pgtbl.h"
 
 #define PRINTK(format, ...) printk("(CPU%ld:) " format, get_cpuid(), ## __VA_ARGS__)
```

Comment:
This should really be in printk.h if we have it or in kernel.h. @yzcode

Reply:
Should I remove it or he do it, for mental consistency?